py/genv.py

```diff
@@ -9,8 +9,25 @@
 from typing import Callable, Dict, Optional, Union
 
 DATETIME_FMT = '%d/%m/%Y %H:%M:%S'
+MEMORY_TO_BYTES_MULTIPLIERS_DICT = {
+    'b': 1,
+    'k': 1000,
+    'm': 1000 * 1000,
+    'g': 1000 * 1000 * 1000,
+    'ki': 1024,
+    'mi': 1024 * 1024,
+    'gi': 1024 * 1024 * 1024,
+}
 
-def poll_pid(pid: int) -> bool:
+
+# cleanup utils
+
+def proc_kill(pid: int) -> bool:
```

Comment:
not sure about this renaming (see my previous comment)
we might want to keep it `poll_pid`, wdyt?

Reply:
I changed the name back.